lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -413,7 +413,7 @@ const expectations = [
       },
     },
   },
-  // TODO: Uncomment when Chrome m84 lands
+  // TODO(COMPAT): Uncomment when Chrome m84 lands
```

Comment:
It'd save me a merge conflict with #10976 if you just skipped this :)

Reply:
true!

but i'mma keep it because the intent it to account for all of these.